### PR TITLE
Improve the sections JS performance by deferring WYSIWYG creation

### PR DESCRIPTION
**Pull request by danielsamuels**

### CODE REVIEW

File: cms/static/cms/js/jquery.cms.wysiwyg.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 django.jQuery(function($){
-    $('.wysiwyg:visible').each(function(){
-        activate_tinymce(this)
-    });
+    setTimeout(() => {
```
Comment (dan-gamble) [inline cms/static/cms/js/jquery.cms.wysiwyg.js]: Should this be `setInterval`?

Comment (danielsamuels) [reply cms/static/cms/js/jquery.cms.wysiwyg.js]: No? I'm just delaying it.

Comment (dan-gamble) [reply cms/static/cms/js/jquery.cms.wysiwyg.js]: How does it render WYSIWYG's that aren't in the viewport on initial page load?

Comment (danielsamuels) [reply cms/static/cms/js/jquery.cms.wysiwyg.js]: Hmm, it doesn't. That's an existing bug it seems - I didn't change the selector, just added a delay.

Comment (danielsamuels) [reply cms/static/cms/js/jquery.cms.wysiwyg.js]: Sorted in onespacemedia/project-template#237.

**Pull request merged**